cmd/root.go

```diff
@@ -102,19 +102,34 @@ func feedViaStdin(ctx context.Context, custom *custom.CustomBouncer, config *cfg
 
 		return c.Wait()
 	}
-	var err error
-	if config.TotalRetries == -1 {
-		for {
-			err = f()
-			log.Errorf("Binary exited: %s", err)
-		}
-	} else {
-		for i := 1; i <= config.TotalRetries; i++ {
-			err = f()
-			log.Errorf("Binary exited (retry %d/%d): %s", i, config.TotalRetries, err)
+
+	if config.TotalRetries == 0 {
```

Comment:
we should set this in pkg/cfg no as it "defaults"?

Reply:
yes